lib/readline.js

```diff
@@ -484,7 +483,7 @@ Interface.prototype._tabComplete = function(lastKeypressWasTab) {
     self.resume();
 
     if (err) {
-      debug('tab completion error %j', err);
+      self._writeToOutput(`tab completion error ${err}`);
```

Comment:
The output is not the same as before anymore. The `%j` called `JSON.stringify()` on the error and caught circular references as well. I suggest to use `util.inspect(err)` in this case.